debian/rules

```diff
@@ -8,8 +8,13 @@ TENSORFLOW_BUILD := no
 endif
 SHELL := /bin/bash
 
+ifeq ($(DEB_HOST_ARCH), amd64)
+CXXFLAGS = -Wno-sign-compare -Wno-unused-but-set-variable -Wno-format-security -Wno-format -fPIC -std=c++11 -msse4.2 -mfpmath=sse
```

Comment:
only `sse2` is part of amd64 specification. With `sse4.2`, I am not sure if this will work with all amd64 arch.